data/json/vehicle_groups.json

```diff
@@ -31,6 +31,10 @@
       [ "policesuv_k9", 5 ],
       [ "police_k9tpt", 1 ],
       [ "truck_swat", 200 ],
+<<<<<<< HEAD
```

Comment:
Bad merge?

Reply:
I have literally no idea..  This is wht I never upload larger projects, *sigh*. Any advice on fixing this so I dont have to upload as a whole new branch or disregard?